tromega/lib/widgets/tracking/interactives/pause_timer.dart

```diff
@@ -24,27 +41,72 @@ class _PauseTimerState extends State<PauseTimer> {
         ),
         child: InkWell(
           onLongPress: () {
```

Comment:
oh lol, ist mir beim testen nicht in den sinn gekommen. an sich cool aber ux technisch ohne hilfe oder press behavior bissl schwer zu finde :D

Reply:
War noch ein artifact der alten Bedienung, ist jetzt mit onTap() besser zu entdecken 👍